app/components/charts/line_chart_component.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= line_chart(
+  reporting_salaries_by_years_of_experience_path,
+  colors: ["#4F46E5", "#14B8A6", "#C026D3"],
+  download: {filename: "salaries_by_years_of_xp"},
+  id: "salary-line-chart",
+  prefix: "$",
+  thousands: ",",
+  xtitle: "Years of Experience",
+  ytitle: "Salary in Dollars"
+) %>
```

Comment:
Can you parameterize the following so that we can pass them into the view component? That way when we make the next line chart in the app, we can re use this more easily.
1. filename
2. prefix (maybe?)
3. xtitle
4. ytitle
5. path

Reply:
Good idea. I'll play with this a bit to allow for easily adjusting and adding options too.